mmv1/third_party/terraform/services/pubsub/resource_pubsub_subscription_test.go

```diff
@@ -238,10 +238,17 @@ func TestAccPubsubSubscriptionBigQuery_serviceAccount(t *testing.T) {
 	topic := fmt.Sprintf("tf-test-topic-%s", acctest.RandString(t, 10))
 	subscriptionShort := fmt.Sprintf("tf-test-sub-%s", acctest.RandString(t, 10))
 
+	if acctest.BootstrapPSARoles(t, "service-", "gcp-sa-pubsub", []string{"roles/bigquery.dataEditor", "roles/bigquery.metadataViewer"}) {
+		t.Fatal("Stopping the test because roles were added to IAM policy.")
+	}
+
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
 		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
 		CheckDestroy:             testAccCheckPubsubSubscriptionDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
```

Comment:
Can you please also add `time` provider to the test [TestAccPubsubSubscriptionBigQuery_update](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/pubsub/resource_pubsub_subscription_test.go#L198), where `testAccPubsubSubscriptionBigQuery_basic` is also called? Thanks.

Reply:
Oh thanks, good call. updated.